engine/math/t/matrix.lisp

```diff
@@ -0,0 +1,259 @@
+(in-package :lbge-test)
+
+
+(deftest add-test
+  (testing "2x2 matrix addition"
+	   (ok (eqm (add (make-float2x2 #(1 2
```

Comment:
1. Tabs in the source code
2. If Rove is not loaded in your running lisp image, and lisp-mode doesn't know what kind of symbol is `deftest`, indentation becomes quite bizarre, as you can see.
But if you call `(ql:quickload :rove)` in your sly session, deftest would be recognized as something like defun or defamcro, and the code will be auto-indented much nicer.

Also, shortcut for indenting the whole sexp is M-q.